Source/MsBuildIntegration/Rhetos.targets

```diff
@@ -0,0 +1,17 @@
+<Project>
+    <UsingTask AssemblyFile="RhetosBuildTask.dll" TaskName="RhetosBuildTask.RhetosBuild" />
+    <Target Name="RhetosBuild" DependsOnTargets="ResolveProjectReferences" BeforeTargets="CoreCompile" Condition="'$(RhetosBuild)'=='True'" Inputs="@(RhetosInput)" Outputs="RhetosSource\**\*;RhetosAssets\**\*;Resources\**\*">
+        <FindUnderPath Files="@(Compile)" Path="RhetosSource">
+            <Output TaskParameter="InPath" ItemName="OldSourceFiles" />
+        </FindUnderPath>
+        <RhetosBuild RhetosBuildExePath="$(MSBuildThisFileDirectory)..\tools\rhetos.exe" ProjectDirectory="$(ProjectDir)." Assemblies="@(_ResolvedProjectReferencePaths)" AssemblyName="$(AssemblyName)" />      

```

Comment:
Are there any options for CustomErrorRegularExpression and CustomWarningRegularExpression?